subprojects/docs/src/samples/webApplication/customized/build.gradle

```diff
@@ -1,4 +1,13 @@
-apply plugin: 'jetty'
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'org.akhikhl.gretty:gretty:1.4.2'
```

Comment:
I am bit concerned that the version of the plugin gets stale. Could we instead just link to the web application getting started guide we are currently writing? https://github.com/gradle/guides/issues/75